### ISSUE TITLE
2.0 no longer working on html template streams

### ISSUE THREAD

**Issue by ivancuric** [tarunc/gulp-jsbeautifier#22]
```javascript
gulp.task('tpl', function() {
  return gulp.src('src/templates/*.html')
    .pipe(plumber())
    .pipe(data(function(file) {
      jsonData = JSON.parse(fs.readFileSync(src.data));
      return jsonData;
    }))
    .pipe(nunjucksRender({
      src: 'src/templates/',
    }))
    .pipe(prettify())
    .on('error', notify.onError({
      message: function(error) {
        return error.message;
      }
    }))
    .pipe(gulp.dest('dist/'))
    .on('end', reload);
});
```
This code used to work in 1.x and is no longer working. What's the issue?

UPDATE:
The reporter gives off the following errors, for every file:
`File type not specified for ajax_autocomplete_city.html`

**Comment by biasso** [comment]
Can you also post the debug messages? (https://github.com/tarunc/gulp-jsbeautifier#debug)

**Comment by tarunc** [comment]
Hey @ivancuric, I published a major update (v2.0.0) to the npm package yesterday but looks like I might have introduced a bug inadvertently.

Thus, I'm removing gulp-jsbeautifier v2.0.0 from npm-- and will look into the bug tonight/try to push out an update. Please try running npm i gulp-jsbeautifier@1.0.2 or npm update and try running your gulp task.
